src/Inciter/Partitioner.h

```diff
@@ -1283,7 +1284,7 @@ class Partitioner : public CBase_Partitioner< HostProxy,
     //! Signal back to host that we are ready for partitioning the mesh
     void signal2host_setup_complete( const CProxy_Transporter& host ) {
       Group::contribute(
-        CkCallback(CkIndex_Transporter::redn_wrapper_partition(NULL), host ));
+        CkCallback(CkIndex_Transporter::redn_wrapper_part(NULL), host ));
```

Comment:
Yes, unfortunately, in this case, I'm afraid, we need the barriers because Zoltan is MPI.